Build failes (obelfilterwidget.cpp: error: ‘FILTER_SCHARR’ was not declared in this scope)

<!--
If you have a question rather than reporting a bug please go to http://answers.opencv.org where you get much faster responses.
If you need further assistance please read [How To Contribute](https://github.com/opencv/opencv/wiki/How_to_contribute).

This is a template helping you to create an issue which can be processed as quickly as possible. This is the bug reporting section for the OpenCV library.
-->

##### System information (version)
- OpenCV => master
- Operating System / Platform => Fedora29 (CUDA 10)
- Compiler => 8.2.1 20181105

##### Detailed description

Building fails with error :
```
opencv_contrib/modules/cvv/src/qtutil/filter/sobelfilterwidget.cpp: In member function ‘virtual void cvv::qtutil::SobelFilterWidget::applyFilter(cvv::qtutil::SobelFilterWidget::InputArray, cvv::qtutil::SobelFilterWidget::OutputArray) const’:
opencv_contrib/modules/cvv/src/qtutil/filter/sobelfilterwidget.cpp:136:11: error: ‘FILTER_SCHARR’ was not declared in this scope
   ksize = FILTER_SCHARR;
           ^~~~~~~~~~~~~
opencv_contrib/modules/cvv/src/qtutil/filter/sobelfilterwidget.cpp:136:11: note: suggested alternative:
In file included from opencv_contrib/modules/cvv/src/qtutil/filter/sobelfilterwidget.cpp:3:
opencv/modules/imgproc/include/opencv2/imgproc.hpp:208:5: note:   ‘FILTER_SCHARR’
     FILTER_SCHARR = -1
     ^~~~~~~~~~~~~
opencv_contrib/modules/cvv/src/qtutil/filter/sobelfilterwidget.cpp: In member function ‘virtual std::pair<bool, QString> cvv::qtutil::SobelFilterWidget::checkInput(cvv::qtutil::SobelFilterWidget::InputArray) const’:
opencv_contrib/modules/cvv/src/qtutil/filter/sobelfilterwidget.cpp:261:11: error: ‘FILTER_SCHARR’ was not declared in this scope
   ksize = FILTER_SCHARR;
           ^~~~~~~~~~~~~
opencv_contrib/modules/cvv/src/qtutil/filter/sobelfilterwidget.cpp:261:11: note: suggested alternative:
In file included from opencv_contrib/modules/cvv/src/qtutil/filter/sobelfilterwidget.cpp:3:
opencv/modules/imgproc/include/opencv2/imgproc.hpp:208:5: note:   ‘FILTER_SCHARR’
     FILTER_SCHARR = -1
     ^~~~~~~~~~~~~
opencv_contrib/modules/cvv/src/qtutil/filter/sobelfilterwidget.cpp:265:15: error: ‘FILTER_SCHARR’ was not declared in this scope
  if (ksize == FILTER_SCHARR)
               ^~~~~~~~~~~~~
opencv_contrib/modules/cvv/src/qtutil/filter/sobelfilterwidget.cpp:265:15: note: suggested alternative:
In file included from opencv_contrib/modules/cvv/src/qtutil/filter/sobelfilterwidget.cpp:3:
opencv/modules/imgproc/include/opencv2/imgproc.hpp:208:5: note:   ‘FILTER_SCHARR’
     FILTER_SCHARR = -1
     ^~~~~~~~~~~~~
At global scope:
cc1plus: warning: unrecognized command line option ‘-Wno-unnamed-type-template-args’
make[2]: *** [modules/cvv/CMakeFiles/opencv_cvv.dir/build.make:440: modules/cvv/CMakeFiles/opencv_cvv.dir/src/qtutil/filter/sobelfilterwidget.cpp.o] Error 1
```

I think that fix it to "cv :: FILTER_SCHARR" or add "using namespace cv;".

--- a/modules/cvv/src/qtutil/filter/sobelfilterwidget.cpp
+++ b/modules/cvv/src/qtutil/filter/sobelfilterwidget.cpp
@@ -12,6 +12,7 @@
 
 namespace cvv
 {
+using namespace cv;
 namespace qtutil
 {
 

